.github/workflows/node.yml

```diff
@@ -38,14 +39,9 @@ jobs:
 
       - name: Install dependencies & build
         run: |
-          make clean
           npm ci
           npm run build --if-present
 
-      - name: npm test
-        run: |
-          npm test
-
```

Comment:
So this would not run the tests anymore?
Then we should at least have another workflow to run the tests.

Reply:
turns out the original template never ran tests. Will update the PR to keep our changes and only update the dependencies.